onnxruntime/contrib_ops/cuda/bert/group_query_attention.cc

```diff
@@ -172,18 +170,31 @@ Status GroupQueryAttention<T>::ComputeInternal(OpKernelContext* context) const {
   if (use_memory_efficient_attention && needs_buff) {
     kv_buffer_bytes = (sizeof(T) * parameters.batch_size * parameters.num_heads * parameters.seqlen_present_kv_cache * parameters.head_size);
   }
+  size_t rotary_buffer_bytes = 0;
+  if (use_memory_efficient_attention && do_rotary_) {
```

Comment:
If memory efficient attention and flash attention are disabled/unused and GroupQueryAttention + RotaryEmbedding fusion is done, will GQA with `do_rotary` enabled be able to run? If not, I think we should document this limitation.

Reply:
there is no unfused gqa implementation at all actually... should i document this detail in bert_defs?

Reply:
Yes, let's document it.